documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -122,6 +122,24 @@ kubectl label secret _CA-KEY-SECRET_ strimzi.io/kind=Kafka strimzi.io/cluster=_C
 * Label `strimzi.io/kind=Kafka` identifies the Kafka custom resource.
 * Label `strimzi.io/cluster=_CLUSTER-NAME_` identifies the Kafka cluster.
 
+. Annotate the secrets
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-CERTIFICATE-SECRET_ strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_
+----
++
+[source,shell,subs="+quotes"]
+----
+kubectl annotate secret _CA-KEY-SECRET_ strimzi.io/ca-key-generation=_CA-KEY-GENERATION_
+----
++
+* Annotation `strimzi.io/ca-cert-generation=_CA-CERTIFICATE-GENERATION_` defines the generation of a new CA certificate.
+* Annotation `strimzi.io/ca-key-generation=_CA-KEY-GENERATION_` defines the generation of a new CA key.
++
+If you are replacing CA certificates automatically generated by the Cluster Operator, use the next higher incremental value from the existing annotation.
```

Comment:
Sorry to be picky. I think the idea to cover the migration from Strimzi generated to custom CA is good and useful. But I think it needs to point to the renewal guide and not suggest to follow the _installing your own CA_ guide. This guide goes through the creation of new secret before deploying cluster. You cannot do that for existing cluster with Strimzi's CA. You would also need to follow the _new key_ renewal pattern to ensure continuity. Just creating a brand new secret with new certs would break the cluster ragardless what generation it uses.

It would also need to include the changes to the Kafka configuration. So maybe it needs separate procedure?

Reply:
I think this is still valid comment.